components/Link.js

```diff
@@ -31,13 +31,7 @@ const Link = ({
   prefetch,
   ...props
 }) => (
-  href.startsWith('http')
-    ? <A {...props} href={href} />
-    : (
-      <NextLink {...props} href={href} prefetch={prefetch}>
-        <A {...props} href={href} />
-      </NextLink>
-    )
+  <A {...props} href={href} />
```

Comment:
Just means you dont get client side routing, every link reloads the page. But nothing will be broken 😀